cypress/e2e/create_tx.cy.js

```diff
@@ -77,6 +79,41 @@ describe('Queue a transaction on 1/N', () => {
     })
     cy.contains('Estimated fee').should('exist')
 
+    // Asserting the relayer option is present
+    cy.contains('Via relayer').find('[type="radio"]').should('be.checked')
+    cy.contains('Sponsored by Gnosis Chain')
+    cy.get('span').contains('Estimated fee').next().should('have.css', 'text-decoration-line', 'line-through')
+    cy.contains('Via relayer').contains(/[0-5] of 5/)
+
+    cy.contains('Estimated fee').click()
+    cy.contains('Edit').click()
+    cy.contains('Owner transaction (Execution)').parents('form').as('Paramsform')
+
+    // Only gaslimit should be editable when the relayer is selected
+    const arrayNames = ['userNonce', 'maxPriorityFeePerGas', 'maxFeePerGas']
```

Comment:
Can we select the inputs from their label? I.e, "Wallet nonce", "Max priority fee (Gwei)" and "Max fee (Gwei)" ?

Reply:
The "name" attribute is already in the inputs, so I don't have to navigate from the label to the input. Is easier to leave it as it is

Reply:
We've already had this discussion in other PRs and the approach we are taking is to target visible text whenever possible. The attribute is not something that is read in the interface.

Reply:
Please see https://github.com/safe-global/web-core/blob/568ea0db252647ecf1524daa8f03e82b0a16883c/cypress/e2e/add_owner.cy.js#L71-L91

Reply:
Ok, I used the labels